### PR TITLE
Add support for new arcconf output format

### PR DESCRIPTION
**Pull request by csware**
Fixes issue #190

### CODE REVIEW

File: lib/App/Monitoring/Plugin/CheckRaid/Plugins/arcconf.pm [hunk 1]
```diff
@@ -464,6 +464,7 @@ sub process_logical_drive_information {
 	shift->process_logical_device_information(@_);
 }
 
+# TODO: pr66/arcconf-getconfig.out, issue31/getconfig, issue190/getconfig
```
Comment (glensc) [inline lib/App/Monitoring/Plugin/CheckRaid/Plugins/arcconf.pm:467]: what does this comment mean? are you working on this? please use github draft status to indicate if the PR is no ready yet

Comment (csware) [reply lib/App/Monitoring/Plugin/CheckRaid/Plugins/arcconf.pm:467]: In an older PR this sub was added, I just added the files in which maxCache is include - to make it easier for someone to work on this lateron.

Comment (csware) [reply lib/App/Monitoring/Plugin/CheckRaid/Plugins/arcconf.pm:467]: I'd work on this, however, I would need getconfig examples where the cache is degregated or resyncing etc.

Comment (csware) [reply lib/App/Monitoring/Plugin/CheckRaid/Plugins/arcconf.pm:467]: This PR is ready to merge for me.

**Pull request merged**